keras_nlp/models/deberta_v3/deberta_v3_tokenizer.py

```diff
@@ -120,6 +122,12 @@ def token_to_id(self, token):
             return self.mask_token_id
         return super().token_to_id(token)
 
+    def detokenize(self, ids):
+        blank_token_id = self.token_to_id("")
+        mask = tf.not_equal(ids, self.mask_token_id)
+        ids = tf.ragged.boolean_mask(ids, mask, default_value=blank_token_id)
```

Comment:
@TheAthleticCoder, your tests are failing because `default_value` is not a valid argument: https://www.tensorflow.org/api_docs/python/tf/ragged/boolean_mask. Please remove `default_value` and trigger the tests again.